src/org/testKitGen/UtilsGen.java

```diff
@@ -40,7 +44,7 @@ public void generateFiles() {
 	}
 
 	private void generateUtil() {
-		try (FileWriter f = new FileWriter(utilsMk)) {
+		try (Writer f = Utility.getWriterObject(arg.getJdkVersion(), arg.getSpec(), utilsMk)) {
```

Comment:
JDK_VERSION is an optional parameter. Please ensure this works if the user does not set JDK_VERSION.